### ISSUE TITLE
MailCatcher UI not working in 0.10.0 (when installed via homebrew)

### ISSUE THREAD

**Issue by iamstuba** [sj26/mailcatcher#560]
Environment:

OS: macOS Sonoma 14.4.1
Ruby: 3.3.2
MailCatcher: 0.10.0
Browser: Google Chrome (125.0.6422.114) and Mozilla Firefox (126.0.1)

<img width="1279" alt="Screenshot 2024-06-04 at 11 32 39" src="https://github.com/sj26/mailcatcher/assets/146061509/1c7d5bcb-57d6-4539-9563-ea1c42aae3fc">

The `.js` and `.css` files from `/assets` are resolving with an incorrect `content-type` of `text/html`, which prevents browsers from executing them.

This issue was not present in the `0.9.0` version of MailCatcher.

**Comment by joshualip-plaudit** [comment]
Were you able to find a solution? (It's happening to me as well)

**Comment by iamstuba** [comment]
Nope. Haven't ever written a line of Ruby, so don't really know how/where the headers are set. I'd hope/think that for anyone more familiar with Ruby and/or MailCatcher this'd be quite an easy fix.

**Comment by kilis** [comment]
Same issue when installed via Homebrew for version 0.10.0

**Comment by iamstuba** [comment]
@joshualip-plaudit @kilis, my current workaround is to just go to `http://127.0.0.1:1080/messages/{X}.html` where `{X}` is the id of the message I want to see. Not optimal, but at least I can work this way. 

Wasn't able to install version `0.9.0` of MailCatcher via Homebrew for some reason.

**Comment by sj26** [comment]
Hm, this is a weird one. And I can't replicate it -- for me they resolve with the correct content type:

<img width="499" alt="image" src="https://github.com/sj26/mailcatcher/assets/14028/b9d7f451-10bd-44be-9d60-da6ed95e6117">

including in Chrome:

<img width="1552" alt="image" src="https://github.com/sj26/mailcatcher/assets/14028/8486b43f-5b7e-4906-b51e-3dcbd9883362">

**Comment by iamstuba** [comment]
```
mailcatcher --foreground
Starting MailCatcher v0.10.0
==> smtp://127.0.0.1:1025
==> http://127.0.0.1:1080

curl -I http://localhost:1080/assets/mailcatcher.js
HTTP/1.1 404 Not Found
X-Cascade: pass
Content-Type: text/html;charset=utf-8
Content-Length: 145
X-XSS-Protection: 1; mode=block
X-Content-Type-Options: nosniff
X-Frame-Options: SAMEORIGIN
Connection: keep-alive
Server: thin
```

Seems like my initial report was incorrect and the `assets` folder isn't even exposed. I've installed MailCatcher through Homebrew.

**Comment by sj26** [comment]
> I've installed MailCatcher through Homebrew.

Ah, interesting. Through the tap? That hasn’t been updated in a very long time.

https://github.com/sj26/homebrew-mailcatcher

**Comment by sj26** [comment]
Oh, there’s an official formula now!

https://github.com/Homebrew/homebrew-core/blob/95f7ea11c1fe7c4f65a81b5e09984cd392316bef/Formula/m/mailcatcher.rb

but it’s missing the step that compiles assets. That’s the issue.

**Comment by iamstuba** [comment]
Seems like the `homebrew` PR was automatically closed? 🤔

**Comment by sj26** [comment]
Yeah. The formula is broken, but I don’t install with homebrew and don’t know how to fix it. Please help if you know how! 🙏

**Comment by james-caresnap** [comment]
@sj26 I opened a [discussion](https://github.com/orgs/Homebrew/discussions/5563) on the homebrew github page to discuss the proper way to fix the homebrew formula.

tldr; it seems like the easiest path forward would be to add a Gemfile.lock to the mailcatcher repo. Then the formula can simply install the released gem version from rubygems.

Any particular feelings about checking in a Gemfile.lock going forward? It seems kind of pointless to me, other than to satisfy the homebrew folks. Unfortunately it lightens the burden on maintaining the homebrew formula by shifting the burden to the mailcatcher repo.

**Comment by james-caresnap** [comment]
@sj26 any thoughts on the above?

**Comment by hermanzamula** [comment]
Somebody please fix this :)

**Comment by aldrienht** [comment]
Having the same issue...

**Comment by sj26** [comment]
I don’t want to add a Gemfile.lock to mailcatcher. That’s an anti pattern for gems. Mailcatcher works with a range of versions. It pushes additional maintenance burden into this repo.

Resolving a set of compatible software versions is the job of the package manager, imho.

**Comment by samirahellab** [comment]
Hi, Anyone found a fix or a work around for this problem? 
I'm not finding a way to install an older version of mailcatcher

**Comment by samirahellab** [comment]
My work around: use a docker container 
docker-compose.yml :
```
version: "3"
 
services:
  mailcatcher:
    restart: always
    image: dockage/mailcatcher:0.9.0
    ports:
    - "1080:1080"
    - "1025:1025"
```
    
  Then run `docker compose up -d `

**Comment by chimmelb** [comment]
> Hi, Anyone found a fix or a work around for this problem? I'm not finding a way to install an older version of mailcatcher

I switched to Mailpit. I know that's not a work around for this repo, but I needed something.

**Comment by joshualip-plaudit** [comment]
Yep. Just switch to Mailpit. It actually works!

**Comment by aydinkazim** [comment]
Same issue, I also switched to mailpit.
